### ISSUE TITLE
malformed node attributes

### ISSUE THREAD

**Issue by joeworkman** [stereobooster/html_press#2]
If you run this following HTML through html_press, you will see that the output gets mangled somehow. I know that this is not 100% valid html. I wanted to get you the smallest code that triggered the bug. 

This html...

```
<embed allowFullScreen="true" allowScriptAccess="always"/>
```

produces this... 

```
<embed allowFullScreen="FS="true" allowScriptAccess="SA="always"/>
```

**Comment by joeworkman** [comment]
I discovered that this issue has to do with the camel case. If I don't use camel case on the attributes, things work ok.

**Comment by stereobooster** [comment]
Yep, I noticed that too. Try to get to it ASAP

**Comment by stereobooster** [comment]
fixed in 0.8.1
